camfeed_centre_x: 320
camfeed_centre_y: 180
pick_height: 10